enterprise/tailnet/pgcoord.go

```diff
@@ -82,7 +84,21 @@ type pgCoord struct {
 // NewPGCoord creates a high-availability coordinator that stores state in the PostgreSQL database and
 // receives notifications of updates via the pubsub.
 func NewPGCoord(ctx context.Context, logger slog.Logger, ps pubsub.Pubsub, store database.Store) (agpl.Coordinator, error) {
-	ctx, cancel := context.WithCancel(ctx)
+	ctx, cancel := context.WithCancel(dbauthz.As(ctx, rbac.Subject{
```

Comment:
I suspect that you learned about the need for RBAC context the hard way...

Reply:
I intellectually knew it, in as much as I helped with the design of dbauthz, but then just forgot when the time came.

Reply:
It would be safer if we can adopt any additional linter to spot these places. cc @johnstcn

Reply:
This is more easily caught by an integration test than a linter.

Reply:
very difficult to spot with a linter, because the subject could be added to the context anywhere along the stack down to the actual DB call.

Reply:
Yea, the current ruleguard linter we use doesn't have the capacity to do that kind of context analysis I don't think. 

Testing should pick this stuff up pretty quick since it should **always** fail if you don't add a subject context.